src/main/java/pl/techlab24/OSKManager/model/validation/ClientValidator.java

```diff
@@ -122,7 +111,7 @@ private static boolean checkSum(String pesel) {
         int[] peselNumbers = new int[11];
 
         for (int i = 0; i < pesel.length(); i++) {
-            peselNumbers[i] = Integer.parseInt(String.valueOf(pesel.charAt(i)));
+            peselNumbers = pesel.chars().map(x -> x - '0').toArray();
```

Comment:
Don't know if it's worth mention, because in my PR this whole method is removed, but just for education purposes - you can replace whole for loop. So - 
instead of  
```
 int[] peselNumbers = new int[11];

        for (int i = 0; i < pesel.length(); i++) {
            peselNumbers[i] = Integer.parseInt(String.valueOf(pesel.charAt(i)));
            peselNumbers = pesel.chars().map(x -> x - '0').toArray();
}
```

you can use 
`int[] peselNumbers = pesel.chars().map(x -> x - '0').toArray();`